Sources/Common/DataModel/ImageData/index.js

```diff
@@ -410,7 +410,7 @@ function vtkImageData(publicAPI, model) {
     }
 
     const average = inum > 0 ? isum / inum : 0;
-    const variance = sumOfSquares - average * average;
+    const variance = Math.abs(sumOfSquares / inum - average * average);
```

Comment:
@dzenanz do you mind reviewing that this is accurate? ty

Reply:
Taking absolute value is not needed, because sum of squares is guaranteed to be larger than square of average. But I believe the formula is now correct.

Reply:
I later realized numerical considerations were probably the reason for that.

Reply:
If we're checking the `inum > 0` case for the average calculation, best to do it here as well.

Reply:
Added check for `inum`, and a test case to cover that possibility.